### ISSUE TITLE
accept full range of CSS values

### ISSUE THREAD

**Issue by chaals** [ThePacielloGroup/CCA-Win#53]
Currently the tool only accepts 6-figure hex values. It would be nice if it accepted the full range specifiable by CSS - 3-figure hex values, rgba(), etc.

**Comment by patrickhlauke** [comment]
i'll note that any alpha values would only make sense for the foreground colour against solid background colour. otherwise, the result would be meaningless as it won't take into account anything else that may be behind the background.

**Comment by patrickhlauke** [comment]
also, using Options > Show Colour Sliders you can get RGB and HSV value entry (though not as a single copy-pasteable input

**Comment by chaals** [comment]
@patrickhlauke 

> i'll note that any alpha values would only make sense for the foreground colour against solid background colour. otherwise, the result would be meaningless as it won't take into account anything else that may be behind the background.

Indeed. Another feature request is to enable the sampling to deal with that, but I **guess** it would be a separate piece of work, and while it may be done at the same time, there is nothing that necessarily requires that to happen in order to produce an improvement to the current capability.
